config/logger/base.yaml

```diff
@@ -17,6 +17,9 @@ test:
   kde:
     bandwidth: 0.1
     kernel: gaussian
+  n_top_k: 5000
+  top_k: 100
+  top_k_period: 2500
```

Comment:
I find this structure a bit confusing, since the top-k metrics variables are children of the `test` config but they are independent in the code. I would say that we either make it completely independent (and follow the same config structure as for `test`) or we merge them with the general `test.` Is there a strong reason to keep them separate? My feeling is that it would be nicer to just merge them (e.g. same period).

Reply:
I kept the structure that was in the code with the `do_*()` scheme. Now wrt the period, that procedure is typically much longer I believe than the regular test is it not? I would _not_ generate 5k samples every 100 steps whild we might want to do other tests.

Reply:
I think most things at test time are similarly expensive or cheap depending on the environment. For example, sampling 5k samples is be a big deal in small environments. A period of 100 is fine for small problems but for bigger problems I use longer periods, like 500.

Reply:
I guess the question, is: do we want the flexibility or not? I would say yes but if you don't it's fine, your call, I'll just increase the shared test period and we'll have fewer data points for those :)